contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+- While an **annual churn of 45%** (aka 5% per month) sounds terrifying, it's not unheard of among successful mid-market B2B SaaS companies. In 2016, Buffer, a successful SaaS company with over $1 million in monthly recurring revenue at the time, reported annual churn of 46%. Multiple studies show a 5% monthly churn rate is common among smaller B2B SaaS companies.
+
+- **Monthly churn above 5%** is to be expected for any early-stage company B2B SaaS, and a good benchmark for B2C subscription products. When you're early-stage, hitting an absolute number is less important than seeing your churn rate improving.
+
+See [more on churn rate benchmarks](#more-on-churn-rate-benchmarks) in the appendix for some of the sources I used to reach these conclusions.
+
+### Problems with churn rate and alternative methods
+
+Other articles you read about churn rate may introduce alternative ways to measure it lifted directly from [this 2011 article](https://shopify.engineering/defining-churn-rate-no-really-this-actually-requires-an-entire-blog-post) by a data scientist at Shopify.
+
+It's a dense affair that highlights some interesting quirks about churn rate, which I'll quickly summarize:
+
+- Extreme growth can lead to distorted churn figures – such as reporting lower churn when user behavior hasn't actually changed. High-growth startups should be wary as a result.
+
+- Because churn rate measures churn among **existing customers**, it can hide churn problems among new, incoming customers.
+
+The Shopify article explores several complex alternative churn formulas to address these problems. It's a laudable objective, but in my view **totally unnecessary**.
+
+Is churn rate a perfect metric? No. Do perfect metrics exist? Also, no. Our objective here isn't mathematical perfection, it's a comparable figure that's easy to understand. The more complicated a metric becomes, the less likely people will use it or understand it. 
+
+Bottom line: You should **never** rely on just one number to measure success. See also why the whole "one number that matters" is a trap.
+
+> 💡**PostHog Tip:** A funnel analysis tracking signup through activation, and conversion-to-paid is a better way to investigate churn among new users. Read our guide to [building funnels in PostHog](/tutorials/funnels) for more on this.
+
+## Understanding retention rate
+
+So that's churn rate, but what about retention rate and why, as I claimed earlier, is it a more useful metric? (most of the time) 
+
+To calculate your overall customer retention over a period you need to know:
+
+- How many customers you have at the start of a period (A)
+- How many customers you have at the end of a period (B)
+- How many new customers were added during that period. (C)
+
+You can then calculate your retention as a percentage (R) using the following formula:
+
+> ((B - C) / A) * 100 = R
```

Comment:
I preferred your churn formula where you used the names of the terms, instead of 'A, B, C' as it's easier to follow